src/v-click-outside.js

```diff
@@ -50,27 +65,39 @@ function bind(el, { value }) {
     return
   }
 
-  el[HANDLERS_PROPERTY] = events.map((eventName) => ({
-    event: eventName,
-    handler: (event) => onEvent({ event, el, handler, middleware }),
-  }))
+  // Note: keep events array immutable, since events value defaults to
+  //       EVENTS variable reference.
+  el[HANDLERS_PROPERTY] = ['vco:faux-iframe-click', ...events].map(
```

Comment:
Should we handle the iframe case separately? If it has the iframe flag, attach only our custom event listener and discard all events and early return.
If it does not have the iframe case, use the standard code? WDYT?

Reply:
🤔  Not sure if I got it correctly, as in, even if we got the `iframe` flag to true we still need to add the other listeners on bind to keep regular behaviour (clicks outside that are not on an iframe?).